client/src/components/TaskListView/TaskListView.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import styles from "./TaskListView.module.css";
+import TaskCard from "../TaskCard/TaskCard";
+import { PropTypes } from "prop-types";
+
+const TaskListView = ({ tasks, taskEdittable }) => {
+  const cards = tasks.map((task, ix) => (
+    <li key={ix} style={{ listStyle: "none" }}>
+      <TaskCard
+        title={task.title}
+        location={task.locationText}
```

Comment:
For the values that are passed in, I would omit `Text` and keep it as `task.endTime` for example. This keeps it more concise and still presents the same information

Reply:
Simplified object attribute names